GUI/Src/pid_tuner_widget.py

```diff
@@ -57,37 +57,49 @@ def __init__(self, pid_tuner_node):
         nav_gui_palette.setColor(self.backgroundRole(), QColor(64, 64, 64))
         self.setPalette(nav_gui_palette)
 
-        self.current_error = 0
-
-        #Create widgets main layout structer
+        #Create widgets main layout structure
         self.primary_linking_layout = QVBoxLayout(self)
         self.setLayout(self.primary_linking_layout)
 
+        #Options widget
         self.options_linking_layout = QGridLayout()
 
         self._error_plotter()
         self._PID_controller_select()
         self._PID_sliders()
+
+        #Set up QTimer to update the PID errors
+        self.pid_error_update_timer = QTimer()
+
+        self.pid_error_update_timer.timeout.connect(lambda: self.pid_gui_node.spinOnce(self.pid_errors_subscriber))
+
         self.primary_linking_layout.addLayout(self.options_linking_layout, 1)
 
-        self.proto_decoder =  Mechatronics_pb2.Mechatronics()
+        #Start PID errors update errors. Update ten timers a second
+        self.pid_error_update_timer.start(10)
```

Comment:
This is 100 times a second. If this is desired I would change the comment to reflect this